package.json

```diff
@@ -9,14 +9,14 @@
     "url": "https://github.com/aditosoftware/eslint-config-adito-platform.git"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^8.41.0",
-    "@typescript-eslint/parser": "^8.41.0",
-    "@stylistic/eslint-plugin": "^5.2.3",
-    "eslint": "^9.34.0",
+    "@typescript-eslint/eslint-plugin": "^8.46.1",
+    "@typescript-eslint/parser": "^8.46.1",
+    "@stylistic/eslint-plugin": "^5.4.0",
+    "eslint": "^9.37.0",
     "eslint-config-prettier": "^10.1.8",
     "eslint-plugin-chai-expect": "^3.1.0",
     "eslint-plugin-chai-friendly": "^1.1.0",
-    "eslint-plugin-jsdoc": "^54.1.1",
+    "eslint-plugin-jsdoc": "^60.8.3",
```

Comment:
Is there a possibility that this version upgrade could cause incompatibility issues in dependent modules?
This is likely to be the case with major upgrades...